subworkflows/local/gtdbtk.nf

```diff
@@ -82,8 +82,10 @@ workflow GTDBTK {
     GTDBTK_SUMMARY (
         ch_filtered_bins.discarded.map{it[1]}.collect().ifEmpty([]),
         GTDBTK_CLASSIFYWF.out.summary.collect().ifEmpty([]),
-        GTDBTK_CLASSIFYWF.out.filtered.collect().ifEmpty([]),
-        GTDBTK_CLASSIFYWF.out.failed.collect().ifEmpty([])
+        [],
+        // GTDBTK_CLASSIFYWF.out.filtered.collect().ifEmpty([]),
+        []
+        // GTDBTK_CLASSIFYWF.out.failed.collect().ifEmpty([])
```

Comment:
```suggestion
        GTDBTK_CLASSIFYWF.out.filtered.collect().ifEmpty([]),
        GTDBTK_CLASSIFYWF.out.failed.collect().ifEmpty([])
```

Still needed, no? I assume this was from debugging

Reply:
In the later/st release of GTDB-Tk, everying is now reported in the summary file.

Reply:
the commetned out bits can be removed (I guess once the module is implemnented :+1: )